### ISSUE TITLE
Drop dependency on debootstrap

### ISSUE THREAD

**Issue by mika** [grml/grml-live#239]
We want to use `mmdebstrap` instead, though it looks like FAI hardcodes usage of `debootstrap`, we should get rid of this.

**Comment by zeha** [comment]
Indeed. We should move the mmdebstrap call from build-driver into grml-live. However I'm not sure how to tell FAI where the base.tar.gz will be.

**Comment by mika** [comment]
IMO let's not add any workarounds for now until we know how to continue with grml-live as of https://github.com/grml/grml-live/issues/267

**Comment by mika** [comment]
Now as of https://github.com/grml/grml-live/pull/281 we support mmdebstrap! \o/
